CHANGES.rst

```diff
@@ -0,0 +1,70 @@
+CHANGES
+=======
+
+0.6 (unreleased)
+----------------
+
+- Add support for Python 3.5 till 3.8.
```

Comment:
Use "through" - "till" is only ever used in conjunction with time.

Reply:
+1, "till" is the informal version of "until." 

The word "till" can also be a noun meaning "a cash register" or a verb meaning "prepare and cultivate land for crops."

Reply:
Yes, but in this context...